corpus/english/135-0.txt

```diff
@@ -1,11 +1,1149 @@
-﻿So long as there shall exist, by virtue of law and custom, decrees of damnation pronounced by society, artificially creating hells amid the civilization of earth, and adding the element of human fate to divine
+
```

Comment:
I think we can delete the lines from 1 to 1013, or not but it can reduce the size of the size

Reply:
But undoubtedly, the english corpus is huge and Good one.